### ISSUE TITLE
Please change app icon

### ISSUE THREAD

**Issue by quartertone** [max-kammerer/orion-viewer#67]
Please change the app icon. When file browsers show the app icon on app-associated files (eg pdfs), the icon looks like a photo preview. Icon should be simpler and easily recognizable as an icon (instead of accidentally looking like a photo)

**Comment by nirodhvana** [comment]
Agreed, it would be fine even if only the book in the logo were bigger
